### ISSUE TITLE
Support absolute paths for ns 

### ISSUE THREAD

**Issue by josesanch** [borkdude/clj-kondo#702]
In shadow-cljs you can use absolute paths for including local files.
This is valid code.
```
(ns main.core
   (:require ["/vendors/daterangepicker"]))
```
clj-kondo raise this error.
java.lang.IllegalArgumentException: /vendors/daterangepicker.transit.json is not a relative path

**Comment by borkdude** [comment]
Hi @josesanch, thanks for posting.

Can you disable the cache as a workaround for now:

```
$ clj-kondo --lint - --cache false <<< '(ns main.core (:require ["/vendors/daterangepicker"]))'
linting took 16ms, errors: 0, warnings: 0
```
